File: CODES/4) Php/9) FUNKSIYA/25.php
<?php
function IsSquare($K)
{
    $a = intval(sqrt($K));
    if (pow($a, 2) == $K) {
        return "true";
    } else {
        return "false";
    }
}

$a = 25;
$b = 36;
$c = 48;
echo $a . " " . IsSquare($a) . "<br>";
echo $b . " " . IsSquare($b) . "<br>";
echo $c . " " . IsSquare($c) . "<br>";

File: CODES/1) C++/9) FUNKSIYA/25.cpp
#include <bits/stdc++.h>

using namespace std;

void IsSquare(int *k){
    int a = sqrt(*k);
	if(pow(a,2)==*k){
		cout<<"True"<<endl;
	}
	else{
		cout<<"False"<<endl;
	}
}
int main(){
	int k;
	for(int i=0; i<3; i++){
		cout << "Son = "; cin >> k;
		IsSquare(&k);
	}
	return 0;
}